src/SIM/SIMbase.C

```diff
@@ -1028,11 +1034,11 @@ bool SIMbase::solveSystem (Vector& solution, int printSol, double* rCond,
 bool SIMbase::solveMatrixSystem (Vectors& solution, int printSol,
                                  const char* compName)
 {
-  solution.resize(myEqSys->getNoRHS());
+  solution.resize(this->getNoRHS());
   for (size_t i = 0; i < solution.size(); i++)
     if (!this->solveSystem(solution[i],printSol,nullptr,compName,i==0,i))
       return false;
-    else
+    else if (solution.size() > 2)
       printSol = 0; // Print summary only for the first solution
```

Comment:
comment seems wrong.

Reply:
Indeed, now it reads "first two solutions" (the second one being the upcoming dual solution)